src/app/cases/page.tsx

```diff
@@ -62,6 +62,7 @@ export default function Page() {
         .filter(
           c =>
             languagesFilters.size === 0 ||
+            c.languages.length === 0 ||
```

Comment:
is there ever a case when there are no languages from cases? isn't language a required field for all listings?

Reply:
cases are pulled from legal server, so it's possible it has missing values, languages is one of them
it's only required for LCA and language support because we can enforce that field to be required on Retool.